features/events/traps/src/main/java/org/opennms/netmgt/trapd/TrapdConfigBean.java

```diff
@@ -81,5 +85,26 @@ public boolean getNewSuspectOnTrap() {
 	public List<SnmpV3User> getSnmpV3Users() {
 		return m_snmpV3Users;
 	}
+	
+	public void onSnmpV3UsersUpdate(TrapdConfiguration config) {
```

Comment:
Please change this method name to onUpdate() and add it to the interface for this class as well.